### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio di ripasso 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#151] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-12 151902](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a4d687bf-1ab2-473c-87e1-0d64dbea976e)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/12f81843-b650-411c-ab0c-d120a5818f9f)
